stylesheets/modules/b2b-account/email-quote.less

```diff
@@ -0,0 +1,34 @@
+#mz-email-modal{
+    position: fixed !important;
+    top: 20% !important;
+    left: 36% !important;
+    width: 35% !important;
+    z-index: 9999 !important;
+}
+#sendEmailText
+{
+  width: 85%;
+  height: 30%;
+  margin-left: 6%;
+}
+.mz-text-margin
+{
+  margin-left: 2.5% !important;
+}
+
+.mz-button-footer {
+    text-align: end;
+}
+.mz-button-width
```

Comment:
don't modify common css classes openly. wrap it around your parent class/container and then modify it. 
if not, then this will reflect in some other places as well where we have used these classes.

see following example-
.your-parent-container{
     .mz-button-width{
            //your styles here
      }
}

Reply:
done